src/components/InstancesList/InstancesList.js

```diff
@@ -872,7 +925,7 @@ class InstancesList extends React.Component {
             icon: 'save',
             messageId: 'ui-inventory.saveInstancesUIIDS',
             onClickHandler: buildOnClickHandler(this.generateInstancesIdReport),
-            isDisabled: isInstancesListEmpty,
+            isDisabled: false // isInstancesListEmpty,
```

Comment:
It looks like it was a temporary solution.

Reply:
Yes, I will use `isRequestUrlExceededLimit` from https://github.com/folio-org/ui-inventory/pull/2540/files#diff-0399c7f70f7645c1cfd8ef8c3b7f167b29a8dc385e86b129f0f46ac896a02144